src/content/add-to-app/ios/project-setup.md

```diff
@@ -5,75 +5,61 @@ description: Learn how to integrate a Flutter module into your existing iOS proj
 ---
 
 Flutter UI components can be incrementally added into your existing iOS
-application as embedded frameworks. There are a few ways to embed Flutter 
-in your existing application.
+application as embedded frameworks.
+To embed Flutter in your existing application,
+consider one of the following three methods.
 
-1. **Use the CocoaPods dependency manager and installed Flutter SDK.**
-  In this case, the `flutter_module` is compiled from
-  the source each time the app is built. (Recommended.)
+| Embedding Method | Methodology | Benefit |
+|---|---|---|
+| Use CocoaPods _(Recommended)_ | Install and use the Flutter SDK and CocoaPods. Flutter compiles the `flutter_module` from source each time Xcode builds the iOS app. | Least complicated method to embed Flutter into your app. |
+| Use [iOS frameworks][] | Create iOS frameworks for Flutter components, embed them into your iOS, and update your existing app's build settings. Run the `flutter build` command to embed the frameworks. | Doesn't require every developer to install the Flutter SDK and Cocoapods on their local machines. |
+| Use iOS frameworks and CocoaPods | Embed the frameworks for your iOS app and the plugins in Xcode, but distribute the Flutter engine as a CocoaPods podspec. Run the `flutter build` command to embed the frameworks. | Provides an alternative to distributing the large `Flutter.xcframework` library. |
```

Comment:
Instead of saying Flutter engine, should we reuse the `Flutter.xcframework` terminology? Using both definitions of the same thing might confuse newbies.

Reply:
Fixed.